integration-cli/docker_cli_rmi_test.go

```diff
@@ -230,50 +206,39 @@ func (s *DockerSuite) TestRmiWithMultipleRepositories(c *check.C) {
 	oldRepo := "busybox"
 	newTag := "busybox:test"
 	out, _, err := dockerCmdWithError("tag", oldRepo, newRepo)
-	if err != nil {
-		c.Fatalf("Could not tag busybox: %v: %s", err, out)
-	}
+	// Could not tag busybox
+	c.Assert(err, checker.IsNil, check.Commentf("out: %s", out))
 
 	out, _, err = dockerCmdWithError("run", "--name", "test", oldRepo, "touch", "/home/abcd")
-	if err != nil {
-		c.Fatalf("failed to run container: %v, output: %s", err, out)
-	}
+	// failed to run container
+	c.Assert(err, checker.IsNil, check.Commentf("out: %s", out))
 
 	out, _, err = dockerCmdWithError("commit", "test", newTag)
-	if err != nil {
-		c.Fatalf("failed to commit container: %v, output: %s", err, out)
-	}
+	// failed to commit container
+	c.Assert(err, checker.IsNil, check.Commentf("out: %s", out))
 
 	out, _, err = dockerCmdWithError("rmi", newTag)
```

Comment:
Yeah, actually it's not a very good idea to put two different kinds of refactor in same PR.
But whatever, I can correct them since we have noticed this problem.

Maybe we can open a new issue for `dockerCmd` modification, that should bring a new long list :smile: 
All up to you :)

Thank you! @HuKeping @vdemeester